pageserver/page_api/src/lib.rs

```diff
@@ -18,6 +18,6 @@ pub mod proto {
     pub use page_service_server::{PageService, PageServiceServer};
 }
 
-mod model;
+pub mod model;
```

Comment:
This isn't necessary, since we import all of the `model` types into the root crate namespace below.